content/live-disk.md

```diff
@@ -32,11 +34,58 @@ Pop!_OS and Ubuntu are remarkably flexible. You can run a full version of Pop!_O
 
 A live disk is a handy tool to have around!
 
+## Verify Your Download
+
+### For Windows
+
+Verifying your download is an important step: we generate a "checksum" for Pop!_OS images and recommend that you verify that your download matches that checksum before trying to install. This ensures that you've received the full, complete download and that it is not corrupted.
+
+Assuming you downloaded Pop!_OS to your Windows Downloads folder, open the Command Prompt in Windows and run the following command:
+
+*For Intel/AMD iso*
+```
+CertUtil -hashfile Downloads\pop-os_21.04_amd64_intel_7.iso sha256
+```
+
+*For NVIDIA iso*
+```
+CertUtil -hashfile Downloads\pop-os_21.04_amd64_nvidia_7.iso sha256
+```
+
+**Note:** The .iso filenames will change overtime so please make sure you are using the correct .iso filename
```

Comment:
`overtime` should be `over time` (two words), there should be a comma before `so`, and this sentence should end with a period.